src/evo/specialtxman.cpp

```diff
@@ -48,9 +48,6 @@ static bool CheckSpecialTxInner(CDeterministicMNManager& dmnman, const Chainstat
         case TRANSACTION_QUORUM_COMMITMENT:
             return llmq::CheckLLMQCommitment(dmnman, chainman, tx, pindexPrev, state);
         case TRANSACTION_MNHF_SIGNAL:
-            if (!DeploymentActiveAfter(pindexPrev, consensusParams, Consensus::DEPLOYMENT_V20)) {
-                return state.Invalid(TxValidationResult::TX_CONSENSUS, "mnhf-before-v20");
-            }
```

Comment:
dcab562e7daf6e199b28e2b74d92b65caf745137: a devnet or something like that could have a mnhf tx get mined before v20 is active though right?

Reply:
https://github.com/dashpay/dash/commit/dcab562e7daf6e199b28e2b74d92b65caf745137 it would not affect mainnet, testnet, indeed.